.github/workflows/releases.yml

```diff
@@ -24,6 +24,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2 
     
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
```

Comment:
python 3.6 no longer receives support. Use 3.8 or 3.9 ?

[table of versions](https://en.wikipedia.org/wiki/History_of_Python#Table_of_versions)

Reply:
use 3.8

Reply:
ok